pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stven_cmd + '?',
+                               set_cmd=stven_cmd
+                               # vals=vals.Numbers(1,4096)
+                               )
+            stvdis_cmd = 'qutech:tvmode{}:disable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_disable'.format(ch_pair),
+                               label=('disable tv mode' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stven_cmd + '?',
+                               set_cmd=stvdis_cmd,
+                               # vals=vals.Bool()
+                               )
+            self.add_parameter('ch_pair{}_tvmode_data'.format(ch_pair),
+                               label=('get TV data ch {} '.format(ch_pair)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getTVdata, ch_pair)
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+            scaladc_cmd = 'qutech:caladc{}'.format(ch_pair)
+            self.add_parameter('ch_pair{}_cal_adc'.format(ch_pair),
+                               label=('Calibrate ADC{}'.format(ch_pair)),
+                               #get_cmd=snsamp_cmd + '?',
+                               set_cmd=scaladc_cmd
+                               # vals=vals.Numbers(1,4096)
+                               )
+            self.add_parameter('ch_pair{}_qstate_cnt_data'.format(ch_pair),
+                               label=('get qstate counter' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getQstateCNT, ch_pair),
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+            self.add_parameter('ch_pair{}_qstate_avg_data'.format(ch_pair),
+                               label=('get qstate average' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getQstateAVG, ch_pair),
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+        for i in range(self.device_descriptor.numChannels):
+            ch = i+1
+            self.add_parameter('ch{}_inavg_data'.format(ch),
+                               label=('get data ch {} '.format(ch)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getInputAverage, ch),
+
+                               vals=vals.Numbers(-128, 127)
+                               )
+            srotres_cmd = 'qutech:wintrot{}:result'.format(ch)
+            self.add_parameter('ch{}_wintrot_result'.format(ch),
```

Comment:
the name `wintrot_result` is a bit confusing for me. Do you think we can have a better name for this? 

Also in this case a docstring would be helpful.